src/app/layout.tsx

```diff
@@ -84,8 +85,14 @@ export default function RootLayout({
 }>) {
   return (
     <html lang="pl" className={poppins.className}>
-      <body className="bg-chat-background">
-        <Providers>{children}</Providers>
+      <body>
+        <Providers>
+          {children}
+          <Script
```

Comment:
i recommend setting data-domains so you don't track visits while developing in localhost :3

https://umami.is/docs/tracker-configuration#data-domains

Reply:
Thanks for advice 🎄🎄